cli.js

```diff
@@ -41,13 +41,13 @@ var cli = meow([
 	'  ava [<file|folder|glob> ...]',
 	'',
 	'Options',
-	'  --init       Add AVA to your project',
-	'  --fail-fast  Stop after first test failure',
-	'  --serial     Run tests serially',
-	'  --require    Module to preload (Can be repeated)',
-	'  --tap        Generate TAP output',
-	'  --verbose    Enable verbose output',
-	'  --no-cache   Disable the transpiler cache',
+	'  --init           Add AVA to your project',
+	'  --fail-fast      Stop after first test failure',
+	'  -s, --serial     Run tests serially',
```

Comment:
I'm not sure I agree. The short flag first keeps everything aligned nicely. 

Long form first means the short flags are scattered instead of in a vertical line. It also means the short flags are creating noise in the middle of what is otherwise readable text